index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body style="height: 100%">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <nav id="navbar">
+      <a href="#home">Home</a>
+      <a href="#work">Work</a>
+      <a href="#contact">Contact</a>
+      <a href="#about">About</a>
+    </nav>
```

Comment:
-It would be nice if your nav tag was wrapped within a semantic tag to help readability and identification

Reply:
Good job 👍  in identifying the `nav` tag always should have to be wrapped in a proper semantic tag. However, it's a kind of vague comment because you suggest the `nav` tag has to be wrapped in the semantic tag but didn't mention which appropriate semantic tag should be used. In this, cause it's good practice to wrap the `nav` tag inside the `header` tag. Please fix that.

Reply:
@Mre55 I thought it was wrong to be suggestive of the solution is reviews. Thanks for the clarification